### PR TITLE
liblzo: Disable unaligned access for arm

### LINKED ISSUE #7602: [Bug]: lzop not working on arm
### Problem description

lzop shows "Illegal instruction" and do not compress file properly

### What steps will reproduce the bug?

install lzop using pkg

then try to compress
```
~ $ du -ha
48K     ./file
52K     .
~ $ lzop file
Illegal instruction
~ $ echo $?
132
~ $ du -ha
48K     ./file
4.0K    ./file.lzo
56K     .
~ $ lzop -t file.lzo
testing file.lzo
lzop: unexpected end of file: file.lzo
```

### What is the expected behavior?

successfully compress the file

### System information

termux-info:

```
Application version:
0.117
Packages CPU architecture:
arm
Subscribed repositories:
# sources.list
deb https://packages.termux.org/apt/termux-main/ stable main
# game-repo (sources.list.d/game.list)
deb https://packages.termux.org/apt/termux-games games stable
# science-repo (sources.list.d/science.list)
deb https://packages.termux.org/apt/termux-science science stable
Updatable packages:
All packages up to date
Android version:
9
Kernel build information:
Linux localhost 4.4.147 #1 SMP PREEMPT Fri Mar 13 14:51:18 CST 2020 armv7l Android
Device manufacturer:
Symphony
Device model:
Symphony G10
```

### PR DESCRIPTION
**Pull request by xtkoba**
Fixes #7602.

### CODE REVIEW

File: packages/liblzo/include-lzo-lzodefs.h.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+--- a/include/lzo/lzodefs.h
++++ b/include/lzo/lzodefs.h
+@@ -2604,7 +2604,8 @@
+ #  define LZO_OPT_UNALIGNED64               1
+ #  endif
+ #elif (LZO_ARCH_ARM)
+-#  if defined(__ARM_FEATURE_UNALIGNED)
++#  if defined(__ANDROID__)
++#  elif 1 && defined(__ARM_FEATURE_UNALIGNED)
```
Comment (Grimler91) [inline packages/liblzo/include-lzo-lzodefs.h.patch:9]: Wouldn't `if defined(__ARM_FEATURE_UNALIGNED) && !defined(__ANDROID__)` be easier?

Comment (xtkoba) [reply packages/liblzo/include-lzo-lzodefs.h.patch:9]: That does not work and results in defining `LZO_OPT_UNALIGNED16` and `LZO_OPT_UNALIGNED32` as `1`.

What follows in the original source is shown below, with the "Boom!" comment added by me.

```c
#elif (LZO_ARCH_ARM)
#  if defined(__ARM_FEATURE_UNALIGNED)
#   if ((__ARM_FEATURE_UNALIGNED)+0)
#    ifndef LZO_OPT_UNALIGNED16
#    define LZO_OPT_UNALIGNED16             1
#    endif
#    ifndef LZO_OPT_UNALIGNED32
#    define LZO_OPT_UNALIGNED32             1
#    endif
#   endif
#  elif 1 && (LZO_ARCH_ARM_THUMB2)          /* Boom! */
#    ifndef LZO_OPT_UNALIGNED16
#    define LZO_OPT_UNALIGNED16             1
#    endif
#    ifndef LZO_OPT_UNALIGNED32
#    define LZO_OPT_UNALIGNED32             1
#    endif
#  elif 1 && defined(__ARM_ARCH) && ((__ARM_ARCH)+0 >= 7)
```

Comment (xtkoba) [reply packages/liblzo/include-lzo-lzodefs.h.patch]: Done.

**Comment by Jdoxka0** [comment]
Cheaks be successful

**Pull request merged**